WallPaper_Project1/WallPaper_Project1/Scenes/MainTabBar/TabBarViewController.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  TabBarViewController.swift
+//  WallPaper_Project1
+//
+//  Created by DuyThai on 07/01/2023.
+//
+
+import UIKit
+
+class TabBarViewController: UITabBarController {
+    let imageViewController = ImageViewController()
+    let videoScreenViewController = VideoScreenViewController()
+    let searchViewController = SearchViewController()
+    let personalViewController = PersonalViewController()
+    let iconNames = ["photo.on.rectangle.angled", "livephoto", "magnifyingglass", "person.fill"]
+    let itemTabBarTitle = ["Photo", "Live Photo", "Search", "Personal"]
+    
```

Comment:
nên dùng enum

Reply:
Mình dùng cái này để gán icon và name cho từng itemTabBar nên mình chưa hình dung được dung enum ntn, tú giải thích mình với

Reply:
tạo enum xong trong enum đó, có 4 case, rồi tạo var icon: String; var title: String, lúc gọi ra chỉ cần .title, .icon

Reply:
Nhưng mà nếu thế thì sử dụng trong vong for như thế nào nhỉ

Reply:
Mình hiểu r nhé, a  Tùng vừa giải thích cho minh r